src/docs/patterns/navigation.mdx

```diff
@@ -1,70 +1,218 @@
+import NavigationTabsSpecsImg from '../../app/assets/design-patterns/navigation/navigation-tabs-specs.png';
+import NavigationBreakdownImg from '../../app/assets/design-patterns/navigation/navigation-breakdown.png';
+import NestedItemImg from '../../app/assets/design-patterns/navigation/nested-item.png';
+import PanelSpecsImg from '../../app/assets/design-patterns/navigation/panel-specs.png';
+// import ProductExample1 from '../../app/assets/design-patterns/navigation/Product Example 1.png';
+// import ContentCover1 from '../../app/assets/design-patterns/navigation/Content Cover 1.png';
+// import ContentCover2 from '../../app/assets/design-patterns/navigation/Content Cover 2.png';
+// import GenericExample1 from '../../app/assets/design-patterns/navigation/Generic Example 1.png';
+// import NestedItemExample from '../../app/assets/design-patterns/navigation/Nested Item Example.png';
+// import NestedItemStandards from '../../app/assets/design-patterns/navigation/Nested Item Standards.png';
+// import ExpandableAccountInfo from '../../app/assets/design-patterns/navigation/Expandable Account Information.png';
+import DividerImage from '../../app/assets/design-patterns/navigation/Divider Image.png';
 
 <!-- KEYWORDS: design pattern navigation drawer menu -->
 
-# Navigation Patterns
+# Navigation
 
-Navigation is the way in which users access the different areas of your application. This is typically accomplished through a side drawer navigation panel with a list of links to the various pages in your application.
+<ImageGrid images={[Banner]} regularWidth />
 
-<MaterialDesignDescription
-    title={'Understanding Navigation'}
-    url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
-    description={`Read about Material Design's navigation fundamentals.`}
-/>
-<MaterialDesignDescription
-    title={'Navigation Drawer '}
-    url={'https://material.io/components/navigation-drawer/'}
-    style={{marginRight: 0}}
+Navigation refers to the way in which users access the different areas of your application and move from screen to screen. The most common forms of primary navigation are drawers and tabs.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Understanding Navigation'}
+        url={'https://material.io/design/navigation/understanding-navigation.html#types-of-navigation'}
+        description={`Read about Material Design's navigation fundamentals.`}
+    />
+    <MaterialDesignDescription
+        title={'Navigation Drawer '}
+        url={'https://material.io/components/navigation-drawer/'}
+        style={{ marginRight: 0 }}
+    />
+    <Divider />
+</React.Fragment>
+
+# Drawer Navigation
+
+A navigation drawer is a panel that is attached on the side of your application. These typically contain a [List](/patterns/lists) of primary pages in your application for the user to access quickly.
+
+<ImageGrid images={[NavigationBreakdownImg]} regularWidth />
```

Comment:
In the image, the divider does not go all the way to the right

The fonts are using gray 500. Should change that to black 500

Reply:
fixed and updated